### DISCUSSION TITLE
have macros directly show up as named entries in the command palette

### DISCUSSION THREAD

**Discussion by jonas-frey** [exceedsystem/vscode-macros#19] [Q&A]
Hi, great extension! Is it possible to have the macros show up in the command palette under custom defined names? Basically I just want shortcuts in the command palette to open certain files ...

**Comment by exceedsystem** [comment]
Hi @jonas-frey. Thanks for using this extension.

Are you referring to the following features?

https://github.com/exceedsystem/vscode-macros/issues/7
